pkg/server/server.go

```diff
@@ -653,6 +663,10 @@ func (s *Server) ListUsers(
 	ctx context.Context,
 	req *openfgav1.ListUsersRequest,
 ) (*openfgav1.ListUsersResponse, error) {
+	if !s.IsExperimentallyEnabled(ExperimentalEnableListUsers) {
+		return nil, fmt.Errorf("ListUsers is not enabled. It can be enabled for experimental use by passing the `--experimentals enable-list-users` configuration option when running OpenFGA server")
```

Comment:
This is throwing a 500.

```suggestion
		return nil, status.Error(codes.Unimplemented, "ListUsers is not enabled. ....
```

Would it be possible to assert on this status code in the test?

Reply:
Good call, was wondering if there was an unimplemented code. I changed that and also added a status code assertion on the test too.